doc/apidoc/classes.rst

```diff
@@ -36,6 +36,9 @@ Bloch sphere
 .. autoclass:: qutip.bloch.Bloch
     :members:
 
+.. autoclass:: qutip.bloch3d.Bloch3d
+    :members:
+
```

Comment:
Alright! I will look into it!

Reply:
Ok I will remove it.
 I did check and was able to build the html docs locally after renaming a file (doc/cqobjevo_compiled_coeff_739974011510.pyx) that couldn't be found while building as the last thee numbers were wrong. I dont know if this might be the issue.

Reply:
It is probably because the docstring of `Bloch3d` includes syntax errors. It wasn't included in API doc so it was hidden.

It is likely caused by https://github.com/qutip/qutip/blob/3eeb22ca747790f80bd8ae3af72713396415cf42/qutip/bloch3d.py#L53
where `>` is a special character in rst. There are 4 of those, corresponding to 4 warnings in the test.

Wrapping the string with backticks may solve this: ` ``['|x>', '']`` `, I think.

Reply:
The reason you may be able to build locally is because in the CI we treat the warnings as errors. You should get the same error we get in the CI if you run `make html SPHINXOPTS="-W --keep-going -T"`.

If we can solve the issue as @BoxiLi suggested we may want to include it in this PR.

Reply:
I tried building locally using `make html SPHINXOPTS="-W --keep-going -T"` and got an error `make: *** [Makefile:45: html] Error 1` even after adding the backticks as @BoxiLi suggested. So I removed the class from the rst file and tried again but the error persisted. The command did seem to build most the html documentation anyway though so I'm quite confused.

Reply:
It seems to run fine now locally for me. Lets not worry too much about this. I will rerun the CI to see if its fixed.